hail/python/test/hail/linalg/test_linalg.py

```diff
@@ -5,12 +5,38 @@
 from ..helpers import *
 import numpy as np
 import tempfile
+import math
 from hail.expr.expressions import ExpressionException
 
 setUpModule = startTestHailContext
 tearDownModule = stopTestHailContext
 
 
+def sparsify_numpy(np_mat, block_size, blocks_to_sparsify):
+    n_rows, n_cols = np_mat.shape
+    target_mat = np.zeros((n_rows, n_cols))
+    n_block_rows = math.ceil(n_rows / block_size)
```

Comment:
for future reference, we usually write this as `(n_rows + block_size - 1) // block_size`